mu/logic.py

```diff
@@ -136,14 +136,19 @@
 logger = logging.getLogger(__name__)
 
 
-def write_and_flush(fd, content):
+def write_and_flush(fileobj, content):
     """
     Writes content to the fd, then flushes and fsyncs to ensure the data is, in
```

Comment:
`Writes content to the fd` -> `Writes content to the fileobj`

Reply:
fixed; thanks